initial commit

SaiAbhinav36 opened the pull request.

SaiAbhinav36 committed:
initial commit

Hrushikesh-2004 committed:
Update ml.py

Added Authentication (login and signup)

SaiAbhinav36 committed:
Delete Frontend/classico/assets/images/Sadie_Sink_2.jpg

HarshithNalla committed:
Add files via upload

Uploaded Mobilenet v3 (vlm model) into the repository.

SaiAbhinav36 committed:
Rename ml.py to imageAndVideoProcessing.py

pranav5069 committed:
Update imageAndVideoProcessing.py

siddhuCodeWiz committed:
Added Llama3 model api

SaiAbhinav36 committed:
Update main.dart

msripooja commented on Backend/imageAndVideoProcessing.py:
```diff
@@ -0,0 +1,154 @@
+mongo = PyMongo(app)
+
+API_URL = "https://api-inference.huggingface.co/models/Salesforce/blip-image-captioning-large"
+HEADERS = {"Authorization": "Bearer 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"}
```
For storing auth headers, API keys use .env file. Sensitive data like API keys, auth headers should never be made public.

msripooja commented on Frontend/classico/README.md:
```diff
@@ -0,0 +1,16 @@
+# classico
```
Modify the default README of this git repo which is at parent directory level. README needs to be elaborate enough to explain the purpose, utility of your project and must provide all instructions to clone, run your project.

msripooja reviewed (changes requested):
Not approving this PR. Please go through my review comments and do the necessary changes.

Hrushikesh-2004 committed:
Update main.dart

I've modified the `LoadConversations` function to enable volunteers to view and respond to conversations. If a response exists, the input field for new responses is hidden, showing the existing response directly.

Hrushikesh-2004 committed:
Update imageAndVideoProcessing.py

I updated the `conversations` endpoint to convert ObjectId into a string for serialization since ObjectId isn't serializable. Additionally, I added a new `updateresponse` endpoint to update the response field of a conversation."

Hrushikesh-2004 committed:
Update imageAndVideoProcessing.py

SaiAbhinav36 committed:
second commit

SaiAbhinav36 committed:
added new files and .env also

SaiAbhinav36 committed:
added new files and .env also

SaiAbhinav36 committed:
Update main.dart

SaiAbhinav36 committed:
added new files and .env also

SaiAbhinav36 committed:
added new files and .env also

SaiAbhinav36 committed:
added new files and .env also

SaiAbhinav36 committed:
Update README.md
